internal/chain/transaction.go

```diff
@@ -131,12 +143,11 @@ func (b *TxBuild) TransferERC20(ctx context.Context, to string, value *big.Int)
 	data = append(data, paddedAddress...)
 	data = append(data, paddedAmount...)
 
-	gasLimit, err := b.client.EstimateGas(ctx, ethereum.CallMsg{
-		To:   &tokenAddress,
-		Data: data,
-	})
-	if err != nil {
-		return emptyHash, err
+	var gasLimit uint64
+	if balance.BitLen() == 0 {
+		gasLimit = 55000
+	} else {
+		gasLimit = 35000
```

Comment:
Can we define constants for these gas limits so that they can be reused within the unit tests as well?